src/GraphQL/Types/Substrate/AttributeType.php

```diff
@@ -31,21 +30,16 @@ public function attributes(): array
     public function fields(): array
     {
         return [
-            // Properties
+            // Computed
             'key' => [
                 'type' => GraphQL::type('String!'),
                 'description' => __('enjin-platform::mutation.batch_set_attribute.args.key'),
+                'alias' => 'key_string',
```

Comment:
Did you try querying this in graphiql? I think this will crash cause this column doesn't exist in DB.